MANIFEST.in

```diff
@@ -1,11 +1,11 @@
 include setup.py
 include AUTHORS
 include LICENSE
-include CHANGELOG
+include CHANGELOG.rst
 include README.rst
+recursive-include cyipopt *.py *.pyx *.pxd *.c
 recursive-include ipopt *.py
-recursive-include src *.pyx *.pxd *.c
-recursive-include test *.py
-recursive-include doc Makefile *.bat *.rst *.py
+recursive-include tests *.py
+recursive-include docs *.bat *.rst *.py
```

Comment:
Is this getting the sphinx makefile?

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/f8c419b3903c8daf8c934f7ae0a40d9ed0b47ce6.